dataactvalidator/interfaces/validatorJobTrackerInterface.py

```diff
@@ -63,22 +63,6 @@ def checkPrerequisites(self, jobId):
 
         return True
 
-    def addStagingTable(self,jobId,stagingTable):
-        """
-
-        Args:
-            jobId: Job to write table name for
-            stagingTable: Name of staging table for this job
-
-        Returns:
-            True if successful
-        """
-        query = self.session.query(Job).filter(Job.job_id == jobId)
-        result = self.checkJobUnique(query)
-        result.staging_table = stagingTable
-        self.session.commit()
-        return True
-
     def checkJobType(self,jobId):
```

Comment:
Good catch removing this, I just pushed a commit to remove the staging_table field from the job table as that is no longer needed.